docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,198 @@
+
+Zeebe, as a workflow engine, is fully **active-active** and replicates data between its brokers. Due to its replication logic, it can handle an entire region failure without data loss. Requiring a proper partition and replication setup. Consider reading through the [cluster technical concept](./../../../components/zeebe/technical-concepts/clustering.md) to learn more about the [Raft protocol](<https://en.wikipedia.org/wiki/Raft_(algorithm)>).
+
+#### Elasticsearch
+
+We treat Elasticsearch as an **active-passive** component. While it may be possible to deploy it in active-active mode and stretch across regions, it's not officially documented by Elasticsearch. Such a setup generally brings its own challenges along that we can't control and may negatively impact the performance of Zeebe.
+
+We go with the approach of having one Elasticsearch per region and configuring Zeebe to export the Elasticsearch data in both regions.
+
+#### WebApps
+
+WebApps are an **active-passive** component and in its current state also not highly available since we're limited by the included exporters that would cause data issues when running multiple instances at the same time.
+
+In every region, there can be only a single WebApp instance running (i.e. in Region 0, 1 instance of Operate, 1 instance of Tasklist; same in Region 1)
+
+This means that one instance will be actively serving traffic, while the other one is on standby.
+
+##### Operate
+
+In the event of a total active region loss, the following data will be lost:
+
+- Batch Operations <!-- Do we have a docs link for this? -->
+
+##### Tasklist
+
+In the event of a total active region loss, the following data will be lost:
+
+- Assignments of tasks
+
+## Requirements
+
+- Camunda 8
+  - Minimum [Helm chart version](https://github.com/camunda/camunda-platform-helm) **9.3+**
+  - Minimum component images
+    - Elasticsearch **8.9+**
+    - Operate **8.5+**
+    - Tasklist **8.5+**
+    - Zeebe **8.5+**
+    - Zeebe Gateway **8.5+**
+- Two Kubernetes clusters
+  - OpenShift is not supported
+  - those need to be able to contact each other via, for example, VPC peering or similar means
+  - an [example implementation](<!-- TODO: Link -->) of two VPC peered Kubernetes clusters based on AWS EKS
+- Elasticsearch usage
+  - OpenSearch (both managed and self-managed) is not supported
+- Maximum RTT (round trip time) of 100 ms between the two Kubernetes clusters
+- Open Ports between the two Kubernetes clusters
+  - **9200** for Elasticsearch since Zeebe has to push data cross-region
+  - **26500** for the Zeebe gateway from client/worker communication
+  - **26501** for the Zeebe brokers and Zeebe gateway communication
+  - **26502** for the Zeebe brokers and Zeebe gateway communication
+- Only specific combinations of Zeebe broker counts and replication factors are supported
+  - `clusterSize` must be a multiple of **two** and a minimum of **four** to evenly distribute the brokers across the two regions
```

Comment:
It's better to spell the numbers consistently as numbers (like we do for the port numbers):
```suggestion
  - `clusterSize` must be a multiple of **2** and a minimum of **4** to evenly distribute the brokers across the two regions
```